### PR TITLE
Decimal precision is inaccurate

### PR DESCRIPTION
**Pull request by range-of-motion**

### CODE REVIEW

**Comment by ryangjchandler** [comment]
All seems pretty good to me @range-of-motion!

File: app/Http/Controllers/ImportController.php [hunk 1]
```diff
@@ -144,7 +145,7 @@ public function postComplete(Request $request, Import $import) {
                     'tag_id' => $row['tag_id'],
                     'happened_on' => $row['happened_on'],
                     'description' => $row['description'],
-                    'amount' => (int) ($amount * 100)
+                    'amount' => Helper::rawNumberToInteger($amount)
```
Comment (Tim-NL) [inline app/Http/Controllers/ImportController.php:148]: The ```rawNumberToInteger``` expects it to be a float, but the $amount seems to not be validated anywhere.

Comment (range-of-motion) [reply app/Http/Controllers/ImportController.php:148]: You mean, we should do that on line 120, with a "numeric" rule?

Comment (Tim-NL) [reply app/Http/Controllers/ImportController.php:148]: Oh, I did not see that one. I think you can also validate on arrays like so; something.*
All things in the "something" array must match the rule in that case.

Comment (range-of-motion) [reply app/Http/Controllers/ImportController.php:148]: Alright, thanks!

**Pull request merged**